org.eclipse.cdt.cross.arc.gnu/plugin.xml

```diff
@@ -679,7 +679,72 @@
             name="Use memory map from TCF"
             valueType="boolean">
       </option>
-
+      <option
+            applicabilityCalculator="com.arc.cdt.toolchain.arc.ArcApplicabilityCalculator"
+            defaultValue="false"
+            id="org.eclipse.cdt.cross.arc.gnu.base.option.target.includecdefines"
+            name="Include C defines from TCF"
+            valueType="boolean">
+      </option>
+      <option
+            applicabilityCalculator="com.arc.cdt.toolchain.arc.ArcApplicabilityCalculator"
+            id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.specsem"
+            name="Use GCC specs file (ARC EM)"
+            valueType="enumerated">
+         <enumeratedOptionValue
+               command="--specs=nsim.specs"
+               id="org.eclipse.cdt.cross.arc.gnu.base.option.linker.nsim"
```

Comment:
It seems to me that value IDs should include option ID as a namespace, e,g. `org.eclipse.cdt.cross.arc.gnu.base.option.linker.specs.nsim`. It seems like those IDs need not be unique, so you can reuse same `org.eclipse.cdt.cross.arc.gnu.base.option.linker.specs.nsim` for both **specs** and **specsem** (I think the don't even need to be `org.eclipse......`, it seems more like a convention, rather than a requirement).